pages/executive/[proposal-id].tsx

```diff
@@ -255,6 +284,11 @@ const ProposalView = ({ proposal }: Props): JSX.Element => {
             <Heading mt={3} mb={2} as="h3" variant="microHeading">
               Supporters
             </Heading>
+            {spellData && (
```

Comment:
should we remove this, since we already show the total MKR support below the title?

Reply:
yes, good catch!